src/components/OSCALProfile.js

```diff
@@ -100,18 +77,31 @@ export default function OSCALProfile(props) {
         </ListSubheader>
       }
     >
-      {!isLoaded
-        ? [...Array(3)].map((_val, index) => renderSkeletonCard(index))
-        : props.profile.resolvedControls.map((control) => (
-            <OSCALControl
-              control={control}
-              includeControlIds={includeControlIds}
-              modificationAlters={props.profile.modify.alters}
-              modificationSetParameters={props.profile.modify["set-parameters"]}
-              childLevel={0}
-              key={`control-${control.id}`}
-            />
-          ))}
+      {isLoaded ? (
+        props.profile.resolvedControls.map((control) => (
+          <OSCALControl
+            control={control}
+            includeControlIds={includeControlIds}
+            modificationAlters={props.profile.modify.alters}
+            modificationSetParameters={props.profile.modify["set-parameters"]}
+            childLevel={0}
+            key={`control-${control.id}`}
+          />
+        ))
+      ) : (
+        <CardContent key="skeleton-card">
+          <span
+            style={{ marginTop: 5, display: "flex", gap: "1em" }}
+            key="controls load 0"
+          >
+            <Skeleton variant="text" width="25em" height="3em" />
+            <Skeleton variant="circle" width="3em" height="3em" />
+          </span>
+          <Skeleton variant="text" width="10em" height="2.5em" />
+          <Skeleton variant="rect" width="100%" height={115} />
```

Comment:
Can we see what it looks like with 2 cards that only contain this 100% width `rect`?